### ISSUE TITLE
Ctrl-Click for <view_type>_view_ref in context on XML form views

### ISSUE THREAD

**Issue by jcfernandez-890825** [trinhanhngoc/pycharm-odoo#43]
<view_type>_view_ref in context can be used to override the default view on relation field definition in form views.

For examples do a search like this:
![image](https://user-images.githubusercontent.com/28876647/140626224-19662ac2-6acb-4de7-be72-9a79baca11cd.png)

**Comment by trinhanhngoc** [comment]
and `kanban_view_ref`:

```xml
<field name="template_id" invisible="1" kanban_view_ref="%(sign.sign_template_view_kanban_mobile)s"/>
```

**Comment by trinhanhngoc** [comment]
This feature will be available in the next release 2021.2.10.
